### DISCUSSION TITLE
New error messages since GO v3.15.1

### DISCUSSION THREAD

**Discussion by kerkovits** [GrandOrgue/grandorgue#2027] [Developers]
Whenever I open the Bureå church sampleset, I get errors like this in GO 3.15.1.

2024. okt. 17., csütörtök, 11:12:01 CEST: 11:12:01: Error: GOSoundStream::ReadBlock: Breaking invalid loop: start_offset=74239, end_pos=74238, new_pos=74239
2024. okt. 17., csütörtök, 11:12:02 CEST: 11:12:02: Error: GOSoundStream::ReadBlock: Breaking invalid loop: start_offset=56319, end_pos=56318, new_pos=56319
2024. okt. 17., csütörtök, 11:12:42 CEST: 11:12:42: Error: GOSoundStream::ReadBlock: Breaking invalid loop: start_offset=56319, end_pos=56318, new_pos=56319

They usually occur a little time after engaging some stops, but before playing any keys.

I have not experienced anything like this before upgrading the version of GO. Since the upgrade, the error is reproducible.

Is the bug in the source code of GO, in the Bureå sampleset, or in my system?

**Comment by oleg68** [comment]
It looks like #1941 when the cash was created before that fix. Recreating the cache with a recent version of GO should eliminate this error.

**Reply by kerkovits** [reply]
I have no cache, I disabled it.

**Comment by chris-lambert-shiels** [comment]
I have a cache - so deleted cache & restarted GO - same error messages.

Could it be an error in the Bureå church sampleset?
